core/Dockerfile

```diff
@@ -8,7 +8,7 @@ RUN apt-get -y update && apt-get install -y curl build-essential fastjar libmagi
 
 ### ADMIN (static build) ###
 WORKDIR /admin
-RUN curl -sL https://github.com/cheshire-cat-ai/admin-vue/releases/download/Admin/develop.zip | jar -xv
+RUN curl -sL https://github.com/cheshire-cat-ai/admin-vue/releases/download/Admin/release.zip | jar -xv
```

Comment:
This should be `develop.zip` in develop branch, and `release.zip` in main branch
If I accept this edit, than I need to manually update main after merging from develop :(
Please take away this edit by branching from develop and PR against develop